### ISSUE TITLE
About the loss function

### ISSUE THREAD

**Issue by Minhvt34** [alimid/Delay-SDE-net#2]
Hi alimid,

Thank you for sharing your code,

I am curious about the max term in the equation (15) in the paper. In the code, `loss_out = criterion(prediction on OOD data, label)`, so, where is the max term?

**Comment by alimid** [comment]
Hi Minhvt34,

I saw that you closed this problem, so I didn't answer it earlier. But adding an answer here now in case others also wonder. You ask about the `loss_out = criterion(prediction on OOD data, label)` where `criterion = nn.BCELoss()` (BCEloss is Binary Cross Entropy between the target and the input probabilities). The BCEloss uses the input data and labels (where labels = 1 for OOD data and 0 for ID data), and wants to predict a probabillity as close to 1 for OOD data a close to 0 for ID data. This means maximizing the probability for OOD data and minimizing it for ID data, just as equation (15) in the paper states. Hope it became clear.
